docs/docs/topics/mutual-auth.md

```diff
@@ -0,0 +1,306 @@
+
+## Pomerium Defaults
+
+> - **Security**: Moderate
+> - **Difficulty**: Easy
+> - **Encrypted**: Yes (from the end user to Pomerium. It's up to the service to provide a TLS endpoint for Pomerium to use.)
+> - **Mutual Authentication**: None
+
+Let's look at a basic installation of Pomerium on a local network, with a single downstream service. This service contains sensitive data that we only want the right people to access.
+
+::: tip Note
+This section describes multiple services of Pomerium like Authenticate and Proxy as separate services. If you are running Pomerium in all-in-one mode, these will be a single process/container in your environment.
+:::
+
+```mermaid
+flowchart TD
+    D[End User]
+        D <--> A
+    subgraph LAN
+    style LAN stroke-dasharray: 5 5
+    A(Pomerium Proxy Service)
+    B(Pomerium Authenticate Service)
+    A <-.-> B
+    C(Service)
+    A <--> C
+    end
+```
+
+1. When the user connects to the route `service.example.com` the traffic is received by the Pomerium Proxy service.
+1. The proxy redirects to the Authenticate service to validate the user by having them sign in with the identity provider (**IdP**).
+1. After authentication succeeds, the proxy verifies that the authenticated (**authn**) user is authorized (**authz**) to access the service, and begins communication with it.
+
+This is a good start, only the users that are supposed to have access to the service get through. But this model is dependent on the security of your network perimeter. If a bad actor gains access to your local area network (**LAN**), they can now communicate with the service directly:
+
+```mermaid
+flowchart TD
+    D[End User]
+    D <--> A
+    E[/Hacker/]
+    E<--->C
+    subgraph LAN
+    style LAN stroke-dasharray: 5 5
+    A(Pomerium Proxy Service)
+    B(Pomerium Authenticate Service)
+    A <-.-> B
+    C(Service)
+    A <--> C
+    end
+```
+
+While your network *should* be secured to only allow traffic at specified ports and directed to specified services, this creates a single point of failure. A hacker need only bypass your firewall to get direct access to your service.
+
+## JWT Verification: Application-based Mutual Authentication
+
+> - **Security**: Strong
+> - **Difficulty**: Moderate
+> - **Encrypted**: Yes (from the end user to Pomerium. It's up to the service to provide a TLS endpoint for Pomerium to use.)
+> - **Mutual Authentication**: Application Layer
+
+Many, but not all, modern web applications support json web tokens (**JWTs**). These tokens are provided by Pomerium (with the [`pass_identity_headers`] key) to the downstream service so that it can independently verify that the traffic it receives is authorized.
```

Comment:
```suggestion
Many, but not all, modern web applications support [json web tokens](https://datatracker.ietf.org/doc/html/rfc7519) (**JWTs**). These tokens can be provided by Pomerium (with the [`pass_identity_headers`] key) so that an upstream service can independently verify that the traffic has properly gone through Pomerium, and that the incoming user request is authorized.
```